### ISSUE TITLE
Feature: "blame" next earlier version of a specific line

### ISSUE THREAD

**Issue by PABourdin** [gitlist-php/gitlist#9]
GitHub has a new feature that I like: When you view  a "blame" page, you may click a button besides the commit hash that directly brings you to the version before that actual line was changed (= the commit before the displayed commit). This is very useful and spares me a lot of time, because I do not need to go to the history, find the previous commit, get the "blame" page, check if the line in question was changed, go back further in time, check again, and so on...

See button "View blame prior to this change" right next to "# years ago":
https://github.com/gitlist-php/gitlist/blame/3dc8401b5e9d5a2c13b1db5896755befc5793966/README.md

**Comment by alehaa** [comment]
Indeed, thats a pretty nice feature. As this doesn't affect the simplicity of GitList I have no concerns about it. However, I think changes in the providers and the routing are required, so we should upgrade to Silex 2 first as described in #4 to avoid modifying the code for the new API.

**Comment by PABourdin** [comment]
I wanted to have a look yesterday and found that I would need "ant" on my webserver. This pulls too many (in particular Java) dependencies that I'm not willing to host on a production system. What is the preferred procedure for GitList development?

**Comment by alehaa** [comment]
@PABourdin I think you could achieve this by using the reference of the blame as starting-point for future blames. E.g.:

```
~:$ git blame README.md | head -n 1
58914c83 (Klaus Silveira   2013-10-23 10:47:31 -0200  1) # GitList: an elegant git repository viewer

~:$ git blame 58914c83^ README.md | head -n 1
^df43c98 (Klaus Silveira   2012-05-18 01:38:33 -0300  1) # GitList: an elegant and modern git repository viewer
```
